js/components/portal-dashboard/student-answers.tsx

```diff
@@ -135,20 +130,20 @@ export class StudentAnswers extends React.PureComponent<IProps> {
     const progressClass = progress > 0 ? css.progress : "";
     return (
       <div className={`${css.activityProgress} ${progressClass}`} key={activity.get("id")}>
-        { this.renderProgressIcon(progress) }
+        { this.renderProgressIcon(progress)}
```

Comment:
Same as above.  I would either go with no space at beginning and end or space at both beginning and end.